### PR TITLE
chore: bump dependencies

### PR DESCRIPTION
**Pull request by alanpoulain**

### CODE REVIEW

File: api/docker/php/docker-entrypoint.sh [hunk 1]
```diff
@@ -55,13 +55,25 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
 			echo "The database is now ready and reachable"
 		fi
 
+		if [ "$APP_ENV" != 'prod' ]; then
+			echo "Create test database"
+			bin/console doctrine:database:create --if-not-exists -e test
```
Comment (vincentchalamon) [inline api/docker/php/docker-entrypoint.sh:60]: Shouldn't it be the purpose of a trait in Behat/PHPUnit?

**Review by vincentchalamon — APPROVED** [review]
LGTM, except for the test environment in the php entrypoint

File: api/docker/php/docker-entrypoint.sh (hunk 1 above)
Comment (alanpoulain) [reply api/docker/php/docker-entrypoint.sh:60]: Even creating the database and loading the fixtures?
Have you already done that somewhere?

Comment (vincentchalamon) [reply api/docker/php/docker-entrypoint.sh:60]: The test database could (should?) be created on demand, the schema and its fixtures could be initialized and loaded thanks to the [RefreshDatabaseTrait](https://github.com/api-platform/demo/blob/main/api/tests/Api/RefreshDatabaseTrait.php)

**Comment by alanpoulain** [comment]
Replaced by https://github.com/api-platform/demo/pull/214.

**Pull request closed without merging**